windows-apps-src/cpp-and-winrt-apis/xaml-cust-ctrl.md

```diff
@@ -203,7 +208,34 @@ struct BgLabelControl : BgLabelControlT<BgLabelControl>
 };
 ```
 
-*Overridable* functions present themselves differently in different language projections. In C#, for example, overridable functions typically appear as protected virtual functions. In C++/WinRT, they're neither virtual nor protected, but you can still override them and provide your own implementation, as shown above.
+*Overridable* functions present themselves differently in different language projections. In C#, for example, overridable functions typically appear as `protected virtual` functions. In C++/WinRT, they're neither `virtual` nor `protected`, but you can still override them and provide your own implementation, as shown above.
+If you are overriding one of these `overridable` functions in C++/WinRT, your `runtimeclass` IDL must not declare the method:
+**IDL**
+```ìdl
+namespace Example
+{
+    runtimeclass CustomVSM : Windows.UI.Xaml.VisualStateManager
+    {
+        CustomVSM();
+        // note that we do not declare GoToStateCore here
+    }
+}
+```
+**C++**
+```cppwinrt
+namespace winrt::Example::implementation
+{
+    struct CustomVSM : CustomVSMT<CustomVSM>
+    {
+        CustomVSM() {}
+
+        bool GoToStateCore(winrt::Windows::UI::Xaml::Controls::Control const& control, winrt::Windows::UI::Xaml::FrameworkElement const& templateRoot, winrt::hstring const& stateName, winrt::Windows::UI::Xaml::VisualStateGroup const& group, winrt::Windows::UI::Xaml::VisualState const& state, bool useTransitions) {
+            return __super::GoToStateCore(control, templateRoot, stateName, group, state, useTransitions);
```

Comment:
Maybe remove the use of the `__super` compiler extension here - cppwinrt provides `base_type` serving the same purpose
```suggestion
            return base_type::GoToStateCore(control, templateRoot, stateName, group, state, useTransitions);
```

Reply:
good idea, thanks